Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -243,8 +243,16 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             if (asyncBreakComplete != null) {
                 asyncBreakComplete(this, new ThreadEventArgs(MainThread));
             }
-        }
-
+        }
+
+        internal bool IsRunning() {
+            var backtraceCommand = new BacktraceCommand(CommandId, _resultFactory, 0, 1);
+            if (TrySendRequestAsync(backtraceCommand).GetAwaiter().GetResult()) {

```

Comment:
This is actually the same code as before, and I wasn't sure why it didn't have the timeout set before, so I just left it as is. I can add a timeout, though.